### PR TITLE
Add avatarview

### PR DESCRIPTION
**Pull request by sthewissen**
## Fixes 
Fixes a small part of #12.

## What changes to the docs does this PR provide?
Adds basic docs for AvatarView.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] For new pages, used the [provided template](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/.template.md).
- [X] For new features, added an entry in the [Table of Contents](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/toc.yml).
- [X] Ran against a spell and grammar checker.
- [X] Contains **NO** breaking changes.

### CODE REVIEW

**Comment by cmcclister** [comment]
Docs Build status updates of commit _[c757392](https://github.com/sthewissen/xamarin-communitytoolkit/commits/c757392c750a0233fd6d5dd710edd1db7a999da9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/sthewissen/xamarin-communitytoolkit/blob/avatarview/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/communitytoolkit/views/avatarview?branch=pr-en-us-19) |
[docs/views/avatarview.md](https://github.com/sthewissen/xamarin-communitytoolkit/blob/avatarview/docs/views/avatarview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/communitytoolkit/views/avatarview?branch=pr-en-us-19) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C12%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202010120932000065-19%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1ZKcQ6AdamZ6Ij2nam66ueTL0GFTojgALhFdMSsEmt8%3D&st=2020-10-12T09%3A27%3A32Z&se=2020-11-12T09%3A32%3A32Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/views/avatarview.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    </StackLayout>
+
+</ContentPage>
+```
+
+## Properties
+
+|Property  |Type  |Description  |
+|---------|---------|---------|
+| BorderColor | [Color](/dotnet/api/xamarin.forms.color) | Gets or sets the border color for the AvatarView. |
```
Comment (davidbritch) [inline docs/views/avatarview.md]: API doc links should use xref syntax for the following reasons:

- They are validated at build time. The format you've used here won't be validated at all.
- They are less to type.

The format is `xref:type`. For example:

```
[`Color`](xref:Xamarin.Forms.Color)
```

Comment (davidbritch) [reply docs/views/avatarview.md]: Note that xrefs are case sensitive.

If in doubt, there's an xref query checker you can use: https://xref.docs.microsoft.com/query?uid=Xamarin.Forms.Color

If you get a response from the query checker, you have the correct type. If you just see [] displayed on the page, you don't have the correct type.

**Review by davidbritch — CHANGES_REQUESTED** [review]
Please switch API doc links to xref format.

**Comment by cmcclister** [comment]
Docs Build status updates of commit _[f0c40e2](https://github.com/sthewissen/xamarin-communitytoolkit/commits/f0c40e2c8d1b97c863d8019ba7e861ca9c602a17)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/sthewissen/xamarin-communitytoolkit/blob/avatarview/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/communitytoolkit/views/avatarview?branch=pr-en-us-19) |
[docs/views/avatarview.md](https://github.com/sthewissen/xamarin-communitytoolkit/blob/avatarview/docs/views/avatarview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/communitytoolkit/views/avatarview?branch=pr-en-us-19) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C16%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202010161144476533-19%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=f3zgDzPgDfT3yItdcx2RSAWPg8uNKhEdrpqC7Tcn488%3D&st=2020-10-16T11%3A40%3A29Z&se=2020-11-16T11%3A45%3A29Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by davidbritch — APPROVED** [review]
Thanks @sthewissen

**Pull request merged**